PayButton/paybutton-server
# Feat/411 3 get blockchain info get block info

This one should be reviewed after #418 

Description:

This PR intends to test the new chronik blockchain client implementation of methods `getBlockchainInfo` and `getBlockInfo`.

Steps to test:

1. open in you browser the networks page: http://localhost:3000/networks (ecash will not show connected because grpc getBlockInfo method seems to be broken for ecash network)
2. Change in constants/index.ts the ecash choice to chronik: `export const NETWORK_BLOCKCHAIN_CLIENTS: KeyValueT<BLOCKCHAIN_CLIENT_OPTIONS> = {   ecash: 'chronik',   bitcoincash: 'grpc' }`
3. Reload the networks page and see ecash status connected
